File: app/src/main/java/bassamalim/hidaya/database/daos/QuizQuestionsDao.java
package bassamalim.hidaya.database.daos;

import androidx.room.Dao;
import androidx.room.Query;

import java.util.List;

import bassamalim.hidaya.database.dbs.QuizQuestionsDB;

@Dao
public interface QuizQuestionsDao {

    @Query("SELECT * FROM quiz_questions")
    List<QuizQuestionsDB> getAll();

}

File: app/src/main/java/bassamalim/hidaya/database/daos/QuizQuestionsDao.kt
package bassamalim.hidaya.database.daos

import androidx.room.Dao
import androidx.room.Query
import bassamalim.hidaya.database.dbs.QuizQuestionsDB

@Dao
interface QuizQuestionsDao {
    @get:Query("SELECT * FROM quiz_questions")
    val all: List<QuizQuestionsDB>
}